test/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ file(GLOB_RECURSE TEST_INCLUDES *.h)
 
 set(TEST_TARGETS
   accessor_tests
+  affinity_tests
```

Comment:
None of these tests will work on Windows, we should either make the test target conditional or have a `SKIP` in every test case.

Reply:
Done. (A few tests should work, i.e. parsing and stringification, so I used skip)